src/test/java/org/primeframework/mvc/GlobalTest.java

```diff
@@ -1448,6 +1448,16 @@ public void post_binary() throws Exception {
                                  .assertHeaderContains("Cache-Control", "no-cache"));
   }
 
+  @Test
+  public void post_binary_no_body() throws Exception {
+    test.simulate(() -> simulator.test("/binary")
+                                 .withURLParameter("expectedNullFile", "true")
+                                 .withContentType("application/octet-stream")
+                                 .post()
+                                 .assertStatusCode(200)
+                                 .assertHeaderContains("Cache-Control", "no-cache"));
```

Comment:
You may want to also assert the body is empty for this test?

```
.assertBodyIsEmpty()
```

Reply:
What would that be testing? The body of the test action response is not important as this assertion, right?

```java
 if (expectedNullFile) {
      assertNull(file);
      return "success";
    }
```

If the "fix" is not in place, then we get a 500.

Unless I'm missing something about how that test action is supposed to work. It seems like due to the `@Status` annotation on the test action, both test methods that call this would be expected to return an empty body (for reasons not directly related to binary stuff?)